keps/sig-network/2593-multiple-cluster-cidrs/README.md

```diff
@@ -855,6 +967,7 @@ high level (needs more precise definitions) those may be things like:
 Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
 implementation difficulties, etc.).
 -->
+TBD
```

Comment:
L963 above - SLOs - please fill in (I'm fine with saying no SLOs) - SLIs are ok here.

And here - if you don't see anything obvious, let's say No.

Reply:
Done, added No